crates/symbolicator/src/services/symbolication.rs

```diff
@@ -571,22 +571,24 @@ fn object_info_from_minidump_module_rust_minidump(
     ty: ObjectType,
     module: &MinidumpModule,
 ) -> RawObjectInfo {
-    let mut code_id = module.code_identifier().into_owned();
+    let mut code_id = module.code_identifier().to_string();
 
     // The processor reports an empty string as code id for MachO files
     // TODO(ja): Fix MinidumpModule::code_identifier for MachO
     if ty == ObjectType::Macho {
-        code_id = module.debug_identifier().unwrap_or_default().into_owned();
-        code_id.truncate(32); // MachO code_id is the debug_id without `0` age.
+        code_id = module
+            .debug_identifier()
+            .map(|id| id.to_string())
+            .unwrap_or_default();
     }
 
     RawObjectInfo {
         ty,
+        // TODO: shouldn't this be None if code_id is empty?
```

Comment:
calling this TODO out since I added this in, maybe I just missed something? maybe `Some("")` is meaningful here?